### ISSUE TITLE
Local test runs busted

### ISSUE THREAD

**Issue by wincent** [graphql/graphql-language-service#145]
Seems like the recent refactoring to use Workspaces/Yarn/Lerna has broken the CI in a few different ways. For example, see https://github.com/graphql/graphql-language-service/issues/139 (about Flow errors).

Top-level `yarn run testonly` is dying locally and in CI with:

```
Error: Cannot find module 'graphql-language-service-utils'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/glh/code/graphql-language-service/packages/interface/src/__tests__/getAutocompleteSuggestions-test.js:17:1)
    ...
```

And you can't run the tests in the sub-dirs as they all have stubs like this defined:

```
    "test": "echo 'Please run `npm test` from the root of the repo' && exit 1",
```

Top-level `build*` scripts are also busted, so I am removing them in https://github.com/graphql/graphql-language-service/pull/144 — we can replace them with something else that actually works.

Seems like we're flying pretty blind at the moment with respect to tests, Flow etc. cc @asiandrummer for visibility.

**Comment by wincent** [comment]
Updated the title because I saw that the tests actually *do* run in Travis CI (example: https://travis-ci.org/graphql/graphql-language-service/builds/265356781), so there is something funky with my local install (this after an `rm -r *` and a clean `git checkout .` and `yarn`).

If I can figure out what is locally wrong with my install, can close this one, but will still need to deal with the other issues (the Flow errors, for example).

**Comment by wincent** [comment]
It's been a while since I worked in this repo and a fair bit has changed since then, so quite possible that i am Doing It Wrong™.

**Comment by asiandrummer** [comment]
`lerna bootstrap --hoist` maybe? And I'm aware of the flow errors with not finding packages correctly - this is a known issue in lerna and can be worked around with including `flow-typed` packages in the top directory. I just didn't find time to do that yet.

**Comment by danez** [comment]
It happened because yarn automatically links all graphql-langauge-service-* package to the local versions and as package.json points to dist/index.js the important thing is to build all packages before running tests. This was fixed in #166 
Now some flow errors appear, but the tests and flow was working when i tried #166, so maybe flow 0.55 changed something.

**Comment by asiandrummer** [comment]
With `flow@0.55.0` there is a bug with Object.create and the flowtype for object prototype not matching. We're waiting for a fix in 0.56.0 and will fix `graphql` package accordingly to properly fix the flow issues, which will mitigate the flow failures in this library (and others such as codemirror-graphql).

**Comment by asiandrummer** [comment]
Tests are passing now - closing this in favor.
